roles/maintenance_10_linux/tasks/main.yml

```
@@ -113,6 +113,40 @@
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
```

10-037 in our checklist has a different name:

```suggestion
    name: "Disk: Logrotate: Are there large files that point to a too verbose service?"
```